public/src/js/models/events.ts

```diff
@@ -83,11 +84,11 @@ export class SecretSantaEvent implements IEvent {
     return `${this.num_participants} Santa Helper${ext}`;
   }
 
-  /** Updated date in human readable format. */
+  /** Updated date in human readable format - relative to current time. */
```

Comment:
just to prevent merge conflicts, might be better to hold off on this. I added something very similar in my giant code review as the event date from the class rather than being hard coded to 25 dec.